docs/content/0.1.0/docs/getting-started/namespaces.md

```diff
@@ -0,0 +1,41 @@
+---
+weight: 15
+---
+
+# Namespaces
+
+All nodes in DataJunction exist within a namespace. Node names are dot separated alpha-numeric elements. The leading elements
+identify the namespace where the node exists. Nodes that do not include any dots in the name are automatically
+defined in the `default` namespace.
+
+| Node Name           | Namespace    |
+|---------------------|--------------|
+| roads.demo.repairs  | roads.demo   |
+| finance.revenue     | finance      |
+| hr.people.employees | hr.people    |
+| customer            | default      |
```

Comment:
So the `default` case is a special one where the namespace won't be included in the node name? Do you think it's worth being consistent and trying to include it in the name or not?

Reply:
We could and probably should, but I like that it's easier to go from 0 to 1 for a new user before they learn about namespaces. Where I think we're going is that there could be an equivalent to `USE default` or `USE roads.demo`.